docs/source/architecture/poet.rst

```diff
@@ -506,13 +506,14 @@ Security Considerations
 -----------------------
 
 1. :math:`T_{WT}` **motivation**: A validator has at most :math:`T_{WT}` seconds
-   to consume a WaitTimer, namely obtain a WaitCertificate on it after the
+   to consume a WaitTimer, namely generate a WaitCertificate on it after the
```

Comment:
Validator doesn't need to run the consensus engine on it. So validator cannot generate a WaitCertificate. `obtain` sounded ok to me. PoET engine will ask SGX enclave to generate one.

Reply:
I agree with you.